BAZProject/ViewModels/MovieViewModel.swift

```diff
@@ -61,7 +73,91 @@ extension MovieViewModel {
         return "\(Calendar.current.component(.year, from: date))"
     }
     
+    /// The vote average of the movie. If the vote average is nil, 0% is returned.
+    /// - Returns: A string representing the vote average of the movie.
     var vote_average: String {
-        return "\((Int(movie.vote_average ?? 0) ) * 10 )% de coincidencia"
+        "\( Int(((movie.vote_average ?? 0) * 10).rounded()) )% de calificación"
+    }
+    
+    var icon_favorite: UIImage? {
+        isFavorite() ? UIImage(systemName: "heart.fill") : UIImage(systemName: "heart")
```

Comment:
Podrías crear también un enum que se encargue de guardar los iconos.
Algo así:

```
/// Enum used to store the icons used through the app.
enum Icon {
    
    static let heart: UIImage = UIImage(systemName: "heart") ?? UIImage()
    static let heartFill: UIImage = UIImage(systemName: "heart.fill") ?? UIImage()
}
```


```suggestion
        isFavorite() ? Icon.heartFill : Icon.heart
```